packages/hardhat-ledger/test/internal/with-spinners.ts

```diff
@@ -1,25 +1,27 @@
+import type { OutputControlledEmitter } from "../../src/internal/with-spinners";
+
 import { assert } from "chai";
 import sinon from "sinon";
 import EventEmitter from "events";
-import * as spinners from "../../src/internal/with-spinners";
+import { withSpinners } from "../../src/internal/with-spinners";
 
 describe("withSpinners", () => {
```

Comment:
Is it hard to add tests for turning off and on events here?

Reply:
I thought about it and I think it is. I'm not sure how can we test the spinner states.